templates/profile/apache/tnsnames.ora.erb

```diff
@@ -0,0 +1,48 @@
+<% @oracle_servers.each do | server, server_alias_arr | -%>
```

Comment:
Ruby programmers like to avoid adding types to variable names (e.g. `_arr`). There are a few reasons (e.g. it adds noise that is more likely to confuse than to inform, it can become misleading as the code is extended), but I think the most important one is that it reveals a variable name which could benefit from more thought.

While it's tempting to be clear with a name like `server_alias_arr`, and `server_aliases` (for example) gives less information, all you need to know happens a couple lines later with

```ruby
server_aliases.each do |server_alias|
```

When we see that, we know that `server_aliases` is something (perhaps a dynamically-generated enumerable, maybe a static array) that can be iterated on (i.e. with `.each`), which is all we need to know. This leaves the door open for us to pass anything that can iterate without potentially lying and claiming that it's an array.

tl;dr: I suggest renaming `server_alias_arr` to `server_aliases`

Reply:
Thanks. :) Makes sense.

Reply:
Done.